.github/workflows/test-release.yaml

```diff
@@ -0,0 +1,55 @@
+name: Test Semantic Release
+
+on: workflow_dispatch
+
+jobs:
+  #  release:
+  #    runs-on: ubuntu-latest
+  #    steps:
+  #      - uses: actions/checkout@v2
+  #        with:
+  #          fetch-depth: 0
+  #          persist-credentials: true
+  #
+  #      - uses: actions/setup-node@v2
+  #        with:
+  #          node-version: '14'
+  #
+  #      - run: yarn install --frozen-lockfile
+  #
+  #      - name: Import GPG key
+  #        uses: crazy-max/ghaction-import-gpg@v3.0.1
+  #        with:
+  #          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+  #          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+  #          git-user-signingkey: true
+  #          git-commit-gpgsign: true
+  #
+  #      - name: Build
+  #        run: yarn run build
+  #
+  #      - run: npx semantic-release
+  #        env:
+  #          GITHUB_TOKEN: ${{ secrets.ADMIN_ACCESS_TOKEN }}
+  #          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+  #          GIT_COMMITTER_NAME: github-actions[bot]
+  #          GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMITTER_EMAIL }}
+  update-tokens:
+    name: Update public JSONBin
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: get tokens data
+        id: tokensData
+        run: cat ./data/tokens.json
+
+      - name: Retrieve data from "working" JSON Bin
+        uses: fjogeleit/http-request-action@v1.8.0

```

Comment:
Did you consider using this action? 
https://github.com/marketplace/actions/jsonbin-action

Reply:
No. I used the same extension to be consisten with fetching data, so anyone who will edit this later will have one, predictable solutions.
But I'll try this one, since it is just easier.